HelpSource/Classes/Object.schelp

```diff
@@ -200,6 +200,113 @@ A String containing the resulting file's path.
 
 subsection::Equality and Identity
 
+Equally asks if the value of two objects is equal.
+Identity asks if they are the same object.
+There are a few classes of object where only one instance exists and is shared between all uses (a singleton).
+These objects include Symbols and numbers.
```

Comment:
Use link::Classes/****:: when mentioning a class

Reply:
OK! but, Symbol and SimpleNumber are classes, but Symbols should be written as link::Symbol::s. It looks not so good, I think.
How about this?
```
These objects include symbols (the instances of link::Classes/Symbol::) and numbers (the instances of the subclasses of link::Classes/SimpleNumber::: link::Classes/Integer:: and link::Classes/Float::).
```

Reply:
I've got no problem with `link::Classes/Symbol::s`.

Reply:
I have seen the case you mentioned several times. I do not like this style...
I can follow your suggestion, but what do you think of the style I wrote?

Reply:
It's fine, but takes up a lot of words. The help files are already incredibly long, they need to be as punchy and concise as possible.